packages/nns/src/canisters/governance/request.converters.ts

```diff
@@ -716,6 +717,21 @@ const fromCommand = (command: ManageNeuronCommandRequest): RawCommand => {
       },
     };
   }
+  if ("SetFollowing" in command) {
+    const { topicFollowing } = command.SetFollowing;
+    return {
+      SetFollowing: {
+        topic_following: topicFollowing.length
```

Comment:
I was wondering if something like `topic_following: toNullable(topicFollowing?.map(etc...))` would not work - i.e. sparing the length check

Reply:
This could work, but `topicFollowing: Array<FolloweesForTopic>` is always defined, so adding a length check helps keep things accurate.